src/colorizer/Dataset.ts

```diff
@@ -147,7 +147,15 @@ export default class Dataset {
     return this.featureNames.includes(name);
   }
 
-  public getFeatureData(name: string): FeatureData {
+  /**
+   * Attempts to get the feature data from this dataset for the given feature name.
+   * Returns `undefined` if feature is not in the dataset.
+   */
+  public tryGetFeatureData(name: string): FeatureData | undefined {
+    return this.features.get(name);
+  }
+
+  private getFeatureData(name: string): FeatureData {
```

Comment:
Nit: something about this makes me think we should get rid of the old one and have the error check always be outside of the function?  I'm assuming this is here for cases where it's ok for the requested feature to be missing. 

I'm not sure about the naming - maybe something like `getFeatureDataOptional` is slightly better than putting `try` in the name of the function.  In reality, the old function is more like a "try" because it can throw something that would be caught.

A minor concern really.